examples/ropsten/README.md

```diff
@@ -0,0 +1,93 @@
+# Chainlink Ropsten Instructions
+
+This guide will allow you to create and deploy a consuming contract to fulfill a data request using our deployed oracle contract. You do not need to run a node yourself in order to follow these instructions.
+
+## Tools
+
+This guide requires the following tools:
+
+- [Metamask](https://metamask.io/)
+- [Remix](https://remix.ethereum.org)
+
+## Setup
+
+Add the Ropsten LINK token to Metamask:
+
+- Switch to the "Ropsten Test Net" network in Metamask
+- Click on the Tokens tab
+- Click Add Token button
+- Paste the contract address 0x20fe562d797a42dcb3399062ae9546cd06f63280
+- The rest should fill in, if it doesn't the Token Symbol is LINK and use 18 for Decimals
+
+You should now see your Ropsten LINK
+
+### Faucets
+
+Ropsten ETH
+- http://faucet.ropsten.be:3001/
+- https://faucet.metamask.io/
+
+Ropsten LINK
+- http...
+
+## Compile Your Consuming Contract
+
+- Update your local repository from [Chainlink](https://github.com/smartcontractkit/chainlink) or [download](https://github.com/smartcontractkit/chainlink/archive/master.zip) a zip.
+- In Remix, import the contracts at `chainlink/examples/ropsten/contracts`
+- Click on the `Consumer.sol` contract in the left side-bar
+- On the Compile tab, click on the "Start to compile" button near the top-right
+- Change to the Run tab
+- Select the Consumer contract on the right side-bar
```

Comment:
Got confused here. Maybe something like `Select Consumer from the dropdown in the right panel".